collect_app/src/main/java/org/odk/collect/android/fragments/FormManagerList.java

```diff
@@ -153,13 +156,24 @@ public void onClick(DialogInterface dialog, int i) {
         alertDialog.show();
     }
 
+    @Override
+    public void progressUpdate(int progress, int total) {
+        String message = String.format(getResources().getString(R.string.deleting_form_dialog_update_message), progress, total);
+        progressDialog.setMessage(message);
+    }
     /**
      * Deletes the selected files.First from the database then from the file
      * system
      */
     private void deleteSelectedForms() {
         // only start if no other task is running
         if (backgroundTasks.deleteFormsTask == null) {
+            progressDialog = new ProgressDialog(getContext());
```

Comment:
We shouldn't be using `ProgressDialog` any longer as [it's deprecated](https://developer.android.com/reference/android/app/ProgressDialog). Annoyingly the warning doesn't show in Android Studio for some reason!

Ideally there would be a different progress indicator that doesn't block the user. Here I don't want to get into that redesign though so for the moment we should just use a `ProgressDialogFragment`.

Reply:
Hi @seadowg! We also use `ProgressDialog` in Delete Saved Forms fragment. Should we remove `ProgressDialog` from there too? Also, I found that the UI would look better if `layout_centerVertical` is set to true in the `progress_dialog.xml` file.